### ISSUE TITLE
HACS - Welcome Screen Shown

### ISSUE THREAD

**Issue by tuga49** [hacs/frontend#100]
Hi,

I'm facing a "problem" with HACS. I've reinstalled HA and I'm now on version 0.108.3. Now I setted up the HACS on yaml that way:
```
hacs:
  token: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
  sidepanel_title: HACS
  sidepanel_icon: "mdi:alpha-c-box"
```
I can use plugins, etc, but every time I restart HA, the HACS show me the welcome screen... The custom components continue works, but it show me the welcome screen every time. I've searched and don't find any issue about this. 

I'm missing something? 

Thank's

![image](https://user-images.githubusercontent.com/17064445/79218596-93c30c80-7e48-11ea-8f2d-ec7ad58a7018.png)

LOG:
```
Error setting up entry for hacs
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 216, in async_setup
    hass, self
  File "/config/custom_components/hacs/__init__.py", line 84, in async_setup_entry
    startup_result = await hacs_startup()
  File "/config/custom_components/hacs/__init__.py", line 158, in hacs_startup
    await setup_frontend()
  File "/config/custom_components/hacs/setup.py", line 103, in setup_frontend
    require_admin=True,
  File "/usr/src/homeassistant/homeassistant/components/frontend/__init__.py", line 190, in async_register_built_in_panel
    raise ValueError(f"Overwriting panel {panel.frontend_url_path}")
ValueError: Overwriting panel hacs
```

**Comment by ludeeus** [comment]
You are missing logs in  your issue.

**Comment by tuga49** [comment]
> You are missing logs in your issue.

Yep, sorry. I added them to issue. Thank's

**Comment by ludeeus** [comment]
You have HACS configured in the UI and with YAML.
Remove the YAML entries and it should work.

This should have been fixed in 0.24.0.....
ProTip "the latest version" is not a version ;)

**Comment by tuga49** [comment]
> You have HACS configured in the UI and with YAML.
> Remove the YAML entries and it should work.
> 
> This should have been fixed in 0.24.0.....
> ProTip "the latest version" is not a version ;)

Problem solved ;) However, I'm using version 0.24.0. Thank's for your help
